youtube_dl/extractor/wrzuta.py

```diff
@@ -80,3 +80,135 @@ def _real_extract(self, url):
             'description': self._og_search_description(webpage),
             'age_limit': embedpage.get('minimalAge', 0),
         }
+
+
+class WrzutaPlaylistIE(InfoExtractor):
+    """
+        this class covers extraction of wrzuta playlist entries
+        the extraction process bases on following steps:
+        * collect information of playlist size
+        * download all entries provided on
+          the playlist webpage (the playlist is split
+          on two pages: first directly reached from webpage
+          second: downloaded on demand by ajax call and rendered
+          using the ajax call response)
+        * in case size of extracted entries not reached total number of entries
+          use the ajax call to collect the remaining entries
+    """
+
+    IE_NAME = 'wrzuta.pl:playlist'
+
+    _VALID_URL = r'https?://(?P<uploader>[0-9a-zA-Z]+)\.wrzuta\.pl/playlista/' \
+                 '(?P<id>[0-9a-zA-Z]+)/.*'
+
+    _TESTS = [{
+        'url': 'http://miromak71.wrzuta.pl/playlista/7XfO4vE84iR/moja_muza',
+        'playlist_mincount': 14,
+        'info_dict': {
+            'id': '7XfO4vE84iR',
+            'title': 'Moja muza',
+        },
+    }, {
+        'url': 'http://noname925.wrzuta.pl/playlista/0uSEHVfbJl2/top_electro_od_noname925',
+        'playlist_mincount': 34,
+        'info_dict': {
+            'id': '0uSEHVfbJl2',
+            'title': 'TOP ELECTRO od noname925',
+        },
+    }, {
+        'url': 'http://panczarek.wrzuta.pl/playlista/52K3cZacAe3/playlista_1',
+        'playlist_mincount': 31,
+        'info_dict': {
+            'id': '52K3cZacAe3',
+            'title': 'playlista #1',
+        },
+    }, {
+        'url': 'http://heroesf70.wrzuta.pl/playlista/6Nj3wQHx756/lipiec_-_lato_2015_muzyka_swiata',
+        'playlist_mincount': 144,
+        'info_dict': {
+            'id': '6Nj3wQHx756',
+            'title': 'Lipiec - Lato 2015 Muzyka Świata',
+        },
+    }, {
+        'url': 'http://andziaoox.wrzuta.pl/playlista/1TOgrSX1hFR/andziaoox_moja_playlista_1',
+        'playlist_mincount': 29,
+        'info_dict': {
+            'id': '1TOgrSX1hFR',
+            'title': 'andziaoox / moja playlista #1',
+        },
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        extracted_id = mobj.group('id')
+        uploader = mobj.group('uploader')
+
+        return self._extracted_playlist_content(
+            url=url,
+            uploader=uploader,
+            playlist_id=extracted_id,
```

Comment:
What's the point of such verbosity? It can be safely written on a single line skipping arg names - they are not keyworded anyway.